ctfcli/core/challenge.py

```diff
@@ -459,12 +460,13 @@ def normalize_requirements(requirements):
     def _normalize_challenge(self, challenge_data: Dict[str, Any]):
         challenge = {}
 
-        copy_keys = ["name", "category", "value", "type", "state", "connection_info"]
+        copy_keys = ["name", "category", "attribution", "value", "type", "state", "connection_info"]
         for key in copy_keys:
             if key in challenge_data:
                 challenge[key] = challenge_data[key]
 
         challenge["description"] = challenge_data["description"].strip().replace("\r\n", "\n").replace("\t", "")
+        challenge["attribution"] = challenge_data["attribution"].strip().replace("\r\n", "\n").replace("\t", "")
```

Comment:
How does this behave if the server doesn't have the attribution field? I feel like it would error when trying to access the attribution field.